src/components/utilities/CippActionsOffcanvas.js

```diff
@@ -49,29 +50,74 @@ export default function CippActionsOffcanvas(props) {
       handleModal(modalMessage, modalUrl, modalType, modalBody)
     }
   }
+
+  const handleOnSelect = (id, url) => {
```

Comment:
don't need to do this, you can pass the value of the selected component by using this.value